src/index.ts

```diff
@@ -26,16 +18,18 @@ class VueBreadcrumbs implements PluginObject<ComponentOptions<Vue>> {
             .flatMap((route: RouteRecord) => {
               let routeRecord: RouteRecord[] = [];
 
-              if (route.meta && route.meta.breadcrumb && route.meta.breadcrumb.parent) {
+              if (route.meta?.breadcrumb?.parent) {
                 const matched = this.$router.resolve({name: route.meta.breadcrumb.parent}).route.matched
 
-                routeRecord = [...matched, ...routeRecord];
+                routeRecord = [...matched];
```

Comment:
You can initially add `route` at `routeRecord`
```diff
- let routeRecord: RouteRecord[] = [];
+ let routeRecord: RouteRecord[] = [route];
```
and add the found here already

```diff
- routeRecord = [...matched];
+ routeRecord = [...matched, ...routeRecord];
```

while retaining the contract

```diff
- return [...routeRecord, route]
+ return routeRecord
```

Reply:
this not changed